### PR TITLE
[GCU E2E test] add stop pfcwd test

### PR DESCRIPTION
**Pull request by isabelmsft**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205

#### What is the motivation for this PR?
Add GCU E2E test for case where we stop pfcwd on specific port
#### How did you verify/test it?
Run E2E test on physical DUT
```
generic_config_updater/test_pfcwd_interval.py::test_stop_pfcwd PASSED    [ 11%]
generic_config_updater/test_pfcwd_interval.py::test_pfcwd_interval_config_updates[True-existing-add] PASSED [ 22%]
generic_config_updater/test_pfcwd_interval.py::test_pfcwd_interval_config_updates[True-existing-replace] PASSED [ 33%]
generic_config_updater/test_pfcwd_interval.py::test_pfcwd_interval_config_updates[True-nonexistent-add] PASSED [ 44%]
generic_config_updater/test_pfcwd_interval.py::test_pfcwd_interval_config_updates[True-nonexistent-replace] PASSED [ 55%]
generic_config_updater/test_pfcwd_interval.py::test_pfcwd_interval_config_updates[False-existing-add] PASSED [ 66%]
generic_config_updater/test_pfcwd_interval.py::test_pfcwd_interval_config_updates[False-existing-replace] PASSED [ 77%]
generic_config_updater/test_pfcwd_interval.py::test_pfcwd_interval_config_updates[False-nonexistent-add] PASSED [ 88%]
generic_config_updater/test_pfcwd_interval.py::test_pfcwd_interval_config_updates[False-nonexistent-replace] PASSED [100%]
```

### CODE REVIEW

**Review by qiluo-msft — DISMISSED** [review]

File: tests/generic_config_updater/test_pfcwd_interval.py [hunk 1]
```diff
@@ -120,13 +120,49 @@ def get_new_interval(duthost, is_valid):
...
+
+    try:
+        tmpfile = generate_tmpfile(duthost)
+        output = apply_patch(duthost, json_data=json_patch, dest_file=tmpfile)
+        expect_op_success(duthost, output)
+        pfcwd_updated_config = duthost.shell("show pfcwd config")
+        pytest_assert(not pfcwd_config['rc'], "Unable to read updated pfcwd config")
+        pytest_assert(interface not in pfcwd_updated_config['stdout'].split(), "pfcwd unexpectedly still running on interface {}".format(interface))
+    finally:
+        delete_tmpfile(duthost, tmpfile)
```
Comment (neethajohn) [inline tests/generic_config_updater/test_pfcwd_interval.py:156]: this whole code needs to be indented. Needs to move under the 'for' block

Comment (isabelmsft) [reply tests/generic_config_updater/test_pfcwd_interval.py:157]: This test only intends to test one interface, as the flow would be the same for all interfaces. I added a break statement to the for loop to stop once the first interface is identified for test

File: tests/generic_config_updater/test_pfcwd_interval.py [hunk 2]
```diff
@@ -120,13 +120,49 @@ def get_new_interval(duthost, is_valid):
...
 
 
+def test_stop_pfcwd(duthost):
+    start_pfcwd = duthost.shell('config pfcwd start_default')
+    pytest_assert(not start_pfcwd['rc'], "Failed to start default pfcwd config")
+    pfcwd_config = duthost.shell("show pfcwd config")
+    pytest_assert(not pfcwd_config['rc'], "Unable to read pfcwd config")
+
+    for line in pfcwd_config['stdout_lines']:
+        if line.startswith('Ethernet'):
```
Comment (qiluo-msft) [inline tests/generic_config_updater/test_pfcwd_interval.py:130]: >line [](http://example.com/codeflow?start=11&length=4)

If there is no `Ethernet` at all, later code will throw.

Comment (isabelmsft) [reply tests/generic_config_updater/test_pfcwd_interval.py:130]: Thanks for the note, I switched to for-else to throw if `interface` is not defined

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**